### PR TITLE
jenkins: Add a script to get the regression report

### PR DESCRIPTION
**Pull request by poornimag**
Add a script that can be run from jenkins to generate regression
report and send the same to gluster-devel

Signed-off-by: Poornima G <pgurusid@redhat.com>

### CODE REVIEW

File: jenkins/scripts/regression-report.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# if anything fails, we'll abort
+set -e
+yum install -y python-dateutil python-blessings python-requests python-lxml
+
+no_of_days=$1
+git clone https://github.com/gluster/glusterfs.git
+cd glusterfs/
+python extras/failed-tests.py --html_report get-summary $no_of_days centos netbsd > regression_report.html
+(
+echo "To: gluster-devel@gluster.org";
```
Comment (poornimag) [inline jenkins/scripts/regression-report.sh]: Need to add "From:" as well. I wasn't sure what would it be.
Also Should this script itself send the mail? or Talur was telling that as Post action of running a job we can send the mail.

File: jenkins/scripts/regression-report.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#Usage: regression-report.sh <last number of days>
+# if anything fails, we'll abort
+set -e
+yum install -y python-dateutil python-blessings python-requests python-lxml
+
+no_of_days=$1
+git clone https://github.com/gluster/glusterfs.git
+cd glusterfs/
```
Comment (raghavendra-talur) [inline jenkins/scripts/regression-report.sh]: use pushd/popd instead

File: jenkins/scripts/regression-report.sh (hunk 1 above)
Comment (raghavendra-talur) [inline jenkins/scripts/regression-report.sh]: I am ok with either of the methods. I would leave it to Nigel.

File: jenkins/scripts/regression-report.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+python extras/failed-tests.py --html_report get-summary $no_of_days centos netbsd > regression_report.html
```
Comment (nigelbabu) [inline jenkins/scripts/regression-report.sh]: Pretty sure `--html_report` is not a valid flag for the command. It's been failing locally for me.

**Comment by nigelbabu** [comment]
I'm going to close this pull request. This report should be generated by https://github.com/nigelbabu/fstat instead.

**Pull request closed without merging**